src/features/views/components/ViewDataTable/columnTypes/SurveyOptionsColumnType.tsx

```diff
@@ -75,18 +75,18 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 const Cell: FC<{
-  cell: SurveyOptionsViewCell;
+  cell: SurveyOptionsViewCell | undefined;
 }> = ({ cell }) => {
   const { orgId } = useRouter().query;
   const styles = useStyles();
   const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
   const { openPane } = usePanes();
 
-  if (cell.length == 0) {
+  if (!cell || cell?.length == 0) {
```

Comment:
I see that the change you made is not what I suggested, which is probably because you got a type error when trying to compare something that may be `undefined` (`cell?.length`) with something that is a number (`0`).

That was my mistake when I wrote my suggestion freehand. You would likely need to do this:

```
if (!cell?.length) {
```

Because that is the equivalent of doing `if (!(cell && cell.length)))`.

If that works, I think it would be nicer, but it doesn't really matter, so I'll merge this now to avoid some other merge conflicts.